src/pages/settings/Wallet/WalletPage/CardDetails.tsx

```diff
@@ -96,7 +87,8 @@ function CardDetails({pan, expiration, cvv, privatePersonalDetails, domain}) {
             />
             <MenuItemWithTopDescription
                 description={translate('cardPage.cardDetails.address')}
-                title={PersonalDetailsUtils.getFormattedAddress(privatePersonalDetails)}
+                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                title={PersonalDetailsUtils.getFormattedAddress(privatePersonalDetails || defaultPrivatePersonalDetails)}
```

Comment:
why not `??`?

Reply:
I'm not sure whether this is right. The thread was closed as resolved, but I don't think this is obvious.

`PrivatePersonalDetails` is a type with all fields being optional. So `{}` is a proper `PrivatePersonalDetails` instance. Do we really want to fall back to `defaultPrivatePersonalDetails` when `privatePersonalDetails` is an empty object?